packages/katello/katello/hostname-change.rb

```diff
@@ -334,7 +334,7 @@ def update_zone(zone, resolver)
       begin
         aaaa_record = resolver.getresource(@old_hostname, Resolv::DNS::Resource::IN::AAAA)
         commands << "update delete #{@old_hostname} AAAA"
-        commands << "update add #{@new_hostname} #{aaaa_record.ttl} A #{aaaa_record.address}"
+        commands << "update add #{@new_hostname} #{aaaa_record.ttl} AAAA #{aaaa_record.address}"
```

Comment:
While this fixes the "original" issue mentioned in the BZ, I think the whole v6 block is not exactly correct.

When we look at the BZ, it tried to add `FE80::5054:FF:FE6C:2A3B` which is a link-local address and should never be in DNS…

Reply:
But then again, this is what the "remote" server answers, so maybe it's not our fault

Reply:
You can detect those `IPAddr.new(aaaa_record.address).link_local?` but I agree that it's out of scope for this PR.